### ISSUE TITLE
"echo" -> "/bin/echo"

### ISSUE THREAD

**Issue by Verdagon** [sheredom/subprocess.h#46]
The reading stdout example in the README didn't work for us; stdout was just empty.

However, when we used "/bin/echo" instead of "echo" for the command, it worked!

I imagine changing the example from echo to /bin/echo would make it not work on windows though, so I'm not sure the wisest course here.

**Comment by sheredom** [comment]
Good catch. The README examples were misleading here: with the default options you should pass an absolute/relative executable path, or explicitly request PATH lookup.

I've updated the examples in `README.md` to use `subprocess_option_search_user_path` with `echo`, and removed the extra quotes around the `Hello, world!` argument.

Fixed in a8fe07b (`Clarify PATH search in README examples`).

---

🧙 Conjured by AI via [pi.dev](https://pi.dev/) using gpt-5.5
